test/fuzz/transformation_replace_id_with_synonym_test.cpp

```diff
@@ -2046,8 +2046,20 @@ TEST(TransformationReplaceIdWithSynonymTest, DISABLED_TypesAreCompatible) {
           %3 = OpTypeFunction %2
           %6 = OpTypeInt 32 1
           %9 = OpTypeInt 32 0
+          %8 = OpTypeStruct %6
+         %10 = OpTypePointer StorageBuffer %8
+         %11 = OpVariable %10 StorageBuffer
+         %12 = OpConstant %6 0
+         %13 = OpTypePointer StorageBuffer %6
+         %15 = OpConstant %6 2
+         %16 = OpConstant %6 7
+         %20 = OpConstant %9 64
           %4 = OpFunction %2 None %3
           %5 = OpLabel
+         %14 = OpAccessChain %13 %11 %12
+         %21 = OpAtomicLoad %6 %14 %15 %20
+         %22 = OpAtomicExchange %6 %14 %15 %20 %16
+               OpAtomicStore %14 %15 %20 %16
```

Comment:
Please improve this, and the rest of the test, to cover all the atomic operations that are allowed with the Shader capability.

Reply:
Done.